src/main/java/catan/domain/transfer/output/game/GameDetails.java

```diff
@@ -38,6 +39,7 @@ public GameDetails(GameBean game, int detailsRequesterId) {
         this.maxPlayers = game.getMaxPlayers();
         this.targetVictoryPoints = game.getTargetVictoryPoints();
         this.currentMove = game.getCurrentMove();
+        this.dice = game.getDiceFirstValue() == null ? null : new DiceDetails(game.getDiceValue(), game.getDiceFirstValue(), game.getDiceSecondValue());
```

Comment:
This changes (dice=null) was discussed with front-end developer before
comit. Moreover, it was his propose:)
And lets leave the format of Details as is, I was changed my opinion.

пʼятниця, 13 листопада 2015 р. користувач andrii-r <notifications@github.com>
написав:

> In src/main/java/catan/domain/transfer/output/game/GameDetails.java
> <https://github.com/is51/catan/pull/34#discussion_r44810704>:
>
> > @@ -38,6 +39,7 @@ public GameDetails(GameBean game, int detailsRequesterId) {
> >          this.maxPlayers = game.getMaxPlayers();
> >          this.targetVictoryPoints = game.getTargetVictoryPoints();
> >          this.currentMove = game.getCurrentMove();
> > +        this.dice = game.getDiceFirstValue() == null ? null : new DiceDetails(game.getDiceValue(), game.getDiceFirstValue(), game.getDiceSecondValue());
>
> Andrii:
> do we need to put dice details when game is not started yet? it is just a
> question for discussion, but we have take a decision and follow it for
> other JSON elements that are not populated yet.
>
> Yurii:
> as temporary decision, I refactored it so that "dice = null" in case if
> dices not thrown yet. IMHO, in future we shoud create separated JSONs for
> new/playing status and preparation/main stage
>
> My questions to this commit:
> 1. Have we already discussed and taken the final decision? This solution
> can make troubles on front-end if it expects not-null value of JSON element
> 'dice' even if dice was not thrown.
> 2. Do you want to disucss format of response in different cases? we have
> already implemented the core and front-end architecture doesn't expect any
> significant changes in response, so if you would like to change the format
> we shuld discuss it as soon as possible but not later.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/is51/catan/pull/34/files#r44810704>.
>